src/InteractiveUI.hs

```diff
@@ -535,29 +536,32 @@ interactiveUI config srcs maybe_exprs = do
 #endif
 
    default_editor <- liftIO $ findEditor
+   current_directory <- liftIO $ getCurrentDirectory
 
    startGHCi (runGHCi srcs maybe_exprs)
-        GHCiState{ progname       = default_progname,
-                   GhciMonad.args = default_args,
-                   prompt         = defPrompt config,
-                   prompt2        = defPrompt2 config,
-                   stop           = default_stop,
-                   editor         = default_editor,
-                   options        = [],
-                   line_number    = 1,
-                   break_ctr      = 0,
-                   breaks         = [],
-                   tickarrays     = emptyModuleEnv,
-                   ghci_commands  = availableCommands config,
-                   last_command   = Nothing,
-                   cmdqueue       = [],
-                   remembered_ctx = [],
-                   transient_ctx  = [],
-                   ghc_e          = isJust maybe_exprs,
-                   short_help     = shortHelpText config,
-                   long_help      = fullHelpText config,
-                   mod_infos      = M.empty,
-                   rdrNamesInScope = []
+        GHCiState{ progname            = default_progname,
```

Comment:
Ahh, the formatting is in a separate commit. But if GitHub is going to merge them all together this way, I'll start doing it in a separate PR. :/